loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts

```diff
@@ -19,6 +19,7 @@ export class KHR_texture_basisu implements IGLTFLoaderExtension {
     public enabled: boolean;
 
     private _loader: GLTFLoader;
+    private _textureInfo: ITextureInfo;

```

Comment:
You can also try storing textureInfo on the ITexture interface. Then you won't need this anymore.